tools/SetupFlow/DevHome.SetupFlow/ViewModels/Environments/ComputeSystemsListViewModel.cs

```diff
@@ -48,8 +48,6 @@ public partial class ComputeSystemsListViewModel : ObservableObject
 
     public AdvancedCollectionView ComputeSystemCardAdvancedCollectionView { get; private set; }
 
-    public Dictionary<DeveloperIdWrapper, ComputeSystemsResult> DevIdToComputeSystemMap { get; set; }
-
     public ComputeSystemProvider Provider { get; set; }
 
     public DeveloperIdWrapper CurrentDeveloperId { get; set; }
```

Comment:
Thats a good call out we'd probably need listen for the event and refresh the page accordingly. I'll create an issue for that since we probably don't want this PR getting too big